lib/miq_automation_engine/service_models/miq_ae_service_vm.rb

```diff
@@ -1,5 +1,7 @@
 module MiqAeMethodService
   class MiqAeServiceVm < MiqAeServiceVmOrTemplate
+    expose :set_remote_console_url
```

Comment:
@gmcculloug 
The DRb.front is an issue and when we have multiple invocations of the Automate Engine running within the same process in different threads like it happens with PUMA, the DRb.front will get overwritten for any thread that kicks off an Automate Method. It might work as long as you have 1 Automate Engine running. We do have 3 variables in the evm.root that can be accessed from the methods directly

$evm.root['user'] 
$evm.root['group']
$evm.root['tenant']

If your goal is to pass the user object into the method you can use the $evm.root approach. If your goal is to use it implicitly inside Service Models you would have to use DRb.front for now till we can come up with a better approach.

The reason all of these is a problem is because DRb could run every request in a different thread and we don't have a way to access the server objects from these threads.